src/main/java/teammates/ui/constants/LogType.java

```diff
@@ -0,0 +1,39 @@
+package teammates.ui.constants;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import teammates.common.util.Const;
+
+/**
+ * Constant values for all the different log types.
+ */
+public enum LogType {
+    // CHECKSTYLE.OFF:JavadocVariable
+    FEEDBACK_SESSION_ACCESS(Const.FeedbackSessionLogTypes.ACCESS),
+    FEEDBACK_SESSION_SUBMISSION(Const.FeedbackSessionLogTypes.SUBMISSION);
+    // CHECKSTYLE.ON:JavadocVariable
+
+    @JsonValue
+    private final String label;
+
+    LogType(String label) {
+        this.label = label;
+    }
+
+    public String getLabel() {
+        return label;
+    }
+
+    /**
+     * Returns the enum value of a log type given its label.
+     */
+    public static LogType valueOfLabel(String label) {
```

Comment:
Just a question here:

If you pass in values like 
`"access", "submission",`
this method returns 
`"FEEDBACK_SESSION_ACCESS", "FEEDBACK_SESSION_SUBMISSION"`
so the latter strings get returned to the frontend, but the frontend currently just expects former. Is there any reason to have the `valueOfLabel`? Since the `generateTypes` command just generates

```
export enum LogType {
  FEEDBACK_SESSION_ACCESS = "access",
  FEEDBACK_SESSION_SUBMISSION = "submission",
}
```

so the comparison is with the "access" and "submission"

Reply:
`valueOfLabel` here should have no effect on the frontend, it's mainly for the backend to be able to convert a given string such as `"access"` to the appropriate `LogType` (such as in the constructor of `FeedbackSessionLogEntryData`)

Reply:
Right. But in the logic of sending back the results, the constructor of `FeedbackSessionLogEntryData` is called so the logType field in that data becomes the name of the label instead of `"access"/"submission"`. So when the response is sent back to frontend, "FEEDBACK_SESSION_ACCESSS" is sent instead of "access". Was thinking that we can just simply send "access" instead of the label name since frontend will(probably) use the "access". Avoids having to do double work of checking for the label instead

Reply:
Would that mean that our API should return a `string` instead of a `LogType` (thus this file can be removed entirely)?
It seems useful though for `LogType` to exist as part of an interface that guarantees our log types are one of only several enumerated options (such as `FEEDBACK_SESSION_ACCESS` or `FEEDBACK_SESSION_SUBMISSION`), rather than a `string` that implies something more open-ended